examples/file-feed/complete/src/browser-app/index.css

```diff
@@ -0,0 +1,24 @@
+@import url('https://fonts.googleapis.com/css?family=Anonymous+Pro|Source+Code+Pro');
```

Comment:
Either we should include the fonts so it won't need to fetch them externally, or we go with the system fonts.